src/sas/system/user.py

```diff
@@ -124,3 +137,140 @@ def copy_old_files_to_new_location():
         if old_path.exists() and not new_path.exists():
             shutil.copy2(old_path, new_path)
 
+
+def module_copytree(module: str, src: PATH_LIKE, dest: PATH_LIKE) -> None:
+    """Copy the tree from a module to the specified directory
+
+    module: name of the Python module (the "anchor" for importlib.resources)
+    src: source name of the resource inside the module
+    dest: destination directory for the resources to be copied into; will be
+        created if it doesn't exist
+    """
+    spth = Path(src)
+    src = str(spth)
+
+    dpth = Path(dest)
+    dpth.mkdir(exist_ok=True, parents=True)
+
+    for resource in importlib.resources.files(module).joinpath(src).iterdir():
+        f_name = dpth / resource.name
+        if "__pycache__" in resource.name:
+            continue
+
+        if resource.is_dir():
+            # recurse into the directory
+            module_copytree(module, spth / resource.name, dpth / resource.name)
+        elif resource.is_file() and not f_name.exists():
+            logging.debug("Copied: %s", spth / resource.name)
+            with open(dpth / resource.name, "wb") as dh:
+                dh.write(resource.read_bytes())
```

Comment:
You might want to wrap this in an exception handler to catch things like permission issues